### PR TITLE
Upgraded core to 0.97.3

### PR DESCRIPTION
**Pull request by cmelchior**
Bumped core to 0.97.3

@realm/java

### CODE REVIEW

**Comment by emanuelez** [comment]
retest this please

**Comment by beeender** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## 0.88.3
 
+* Updated Realm Core to 0.97.3.
```
Comment (emanuelez) [inline CHANGELOG.md]: shouldn't we list what the update fixes?

Comment (cmelchior) [inline CHANGELOG.md]: The fixes all seem to be minor and doesn't effect us from what I could tell.

**Comment by dalinaum** [comment]
:+1:

**Pull request merged**